### ISSUE TITLE
Request: Option for regions

### ISSUE THREAD

**Issue by TitanKaempfer** [Brian-Mp16/Animal-Crossing-Town-Ticket-Station#4]
Is it possible to add the option for the different regional versions of the game in the program?
While quick codes can be redeemed independently from the games version, normal codes sadly are only generated for the US version, making it impossible for them to be redeemed in a PAL/European version.

**Comment by Jacob-Tate** [comment]
I think the code generation is the same for all english versions of the game is this not the case?

**Comment by benduckwdesign** [comment]
@TitanKaempfer I'm willing to dismiss this as a myth based on my own testing, but we're going to need your player and town name, as well as the items you were trying to redeem. From what I tested using player name and town name both as "!" the codes worked the same across both versions. So this may be a problem with a specific item or text character.

**Comment by TitanKaempfer** [comment]
The problem didn't actually happen to me, but a person that I moderate for on Twitch. Trying to look into the VOD to proof, I noticed that she perhaps accidentially added a space in front of her name ingame. So I'm gonna apologize for this mistake leading to a wrong feature report. The game still accepted the quick codes, if available, and online research resulted in telling me that those are the same for all regions, which is why I thought that normal codes might differ. (They may still do but I can't tell for sure, she uses a German copy, if this changes anything at all)

We would have to do some testing there again with the name "  Lully" and "eepytown" and redeeming a fishing rod.

This might however change my suggestion to another thing, if possible: If the player changes/corrects the town or player name afterwards in the program, the program doesn't seem to generate the new codes for items already picked. Because we also suggested in chat that she perhaps accidentially added spaces somewhere in the name and she changed it on stream in the program, with the code for this fishing rod staying the same one that Tom Nook told her is for someone else.
